src/core.js

```diff
@@ -606,12 +606,16 @@ $.extend( $.validator, {
 			return $( this.settings.errorElement + "." + errorClass, this.errorContext );
 		},
 
-		reset: function() {
+		resetLists: function() {
```

Comment:
I fully agree (banged my head against the wall for 20 minutes trying to name this). Any suggestions?